docs/pages/machine-id/manifesto.mdx

```diff
@@ -0,0 +1,52 @@
+---
+title: Machine ID Manifesto
+description: A manifesto for Machine Identity
+---
+
+As time goes on, machines and services are doing more and more impactful and
+privileged work. Services are increasingly communicating with each other across
+network and company boundaries. AI agents are getting increasingly common and
+are becoming more autonomous and powerful at an accelerating pace.
+
+On the other side, services and machines are increasingly a target of attacks,
```

Comment:
We sure do say `increasingly` a lot here..

Reply:
You may say we use it increasingly throughout the section... I agree - I'll swap this out to read a little nicer.

Reply:
80f34d1a73